app/cdap/components/WrangleHome/Components/WrangleCard/index.tsx

```diff
@@ -43,6 +44,11 @@ export default function WrangleCard() {
   // then using unshift function to add an object for Imported Dataset to entire ConnectorTypes Array.
   const getConnectorTypesNames = async () => {
     let fetchedConnectorTypes = await fetchConnectors();
+    const categorizedConnections = await getCategorizedConnections();
+    const connectorTypeWithConnections = [];
+    categorizedConnections.forEach((itemEach, key) => {
+      connectorTypeWithConnections.push(key);
```

Comment:
Why not make `connectorTypeWithConnections` an Object or Map and save on the `find` on line 130?

Reply:
Done